.changeset/tame-apricots-roll.md

```diff
@@ -0,0 +1,6 @@
+---
+'@openfn/language-intuit': major
```

Comment:
changeset isn't needed for new adaptors - please delete this file and then we can merge

Reply:
Done